### ISSUE TITLE
Additional props for keyboard helpers (input components with "hidden accessible" elements)

### ISSUE THREAD

**Issue by Xerphis** [primefaces/primereact#1852]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

**Current behavior**
I cannot add extra props for input components that uses a hidden input field (for accessible - "p-hidden-component").
This issue applies to all input components using hidden accessibility: dropdown, multiselect, checkbox, select etc.

**Expected behavior**
Possibility to add props to the "hidden accessible" input element.

**Minimal reproduction of the problem with instructions**
Why do I need this? I use [@q42/floating-focus-a11y](https://www.npmjs.com/package/@q42/floating-focus-a11y) for better focusing focusable elements. When we focus on an element that uses "hidden accessibility", the hidden element will be highlighted.
![before](https://user-images.githubusercontent.com/13980568/110217301-8c1eb300-7eb3-11eb-916a-e27df3602534.png)
In this case, the documentation suggests adding the attribute "focus-target" to the hidden element and indicating the identifier of the element that should be selected.
```JSX
// primereact/components/dropdown/Dropdown.js, line 785
renderKeyboardHelper() {
	return (
		<div className="p-hidden-accessible">
			<input
				ref={el => (this.focusInput = el)}
				id={this.props.inputId}
				type="text"
				readOnly
				aria-haspopup="listbox"
				onFocus={this.onInputFocus}
				onBlur={this.onInputBlur}
				onKeyDown={this.onInputKeyDown}
				disabled={this.props.disabled}
				tabIndex={this.props.tabIndex}
				aria-label={this.props.ariaLabel}
				aria-labelledby={this.props.ariaLabelledBy}
				focus-target={this.id}
			/>
		</div>
	);
}
```
![after](https://user-images.githubusercontent.com/13980568/110217302-8fb23a00-7eb3-11eb-9027-fff32c63fd4a.png)
The feature can be solved by adding prop "keyboardHelperProps" for pass props or "renderKeyboardHelper".
Suggest:
```JSX
renderKeyboardHelper() {
	let keyboardHelperProps = {
		ref: el => (this.focusInput = el),
		id: this.props.inputId,
		type: "text",
		readOnly: true,
		"aria-haspopup": "listbox",
		onFocus: this.onInputFocus,
		onBlur: this.onInputBlur,
		onKeyDown: this.onInputKeyDown,
		disabled: this.props.disabled,
		tabIndex: this.props.tabIndex,
		"aria-label": this.props.ariaLabel,
		"aria-labelledby": this.props.ariaLabelledBy
	};
	if (this.props.renderKeyboardHelper) {
		return this.props.renderKeyboardHelper(keyboardHelperProps);
	}
	return (
		<div className="p-hidden-accessible">
			<input
				{...keyboardHelperProps}
			/>
		</div>
	);
}
// or
renderKeyboardHelper() {
	return (
		<div className="p-hidden-accessible">
			<input
				...
				{...this.props.keyboardHelperProps}
			/>
		</div>
	);
}
```

**Please tell us about your environment:**
Windows, npm

* **React version:**
17

* **PrimeReact version:**
6.0.2

* **Browser:**
all
 
* **Language:**
all
